app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkController.kt

```diff
@@ -169,7 +169,10 @@ class DefaultBookmarkController(
     override fun handleRequestSync() {
         scope.launch {
             store.dispatch(BookmarkFragmentAction.StartSync)
-            activity.components.backgroundServices.accountManager.syncNow(SyncReason.User)
+            activity.components.backgroundServices.accountManager.syncNow(
+                reason = SyncReason.User,
+                debounce = true,
```

Comment:
I'm not sure if this is the right patch to be changing the syncing of bookmarks, but at the very least, if you want to include this patch, let's maybe take out the debounce and use `customEngineSubset = listOf(SyncEngine.Bookmarks)`?

The concern with the home screen is because it is a hot path for users and we are using more resources there as a result whenever they use that flow. We haven't evaluated if bookmarks/history has similar concerns.

Reply:
I see.  Sure, I'll update and update to use the bookmark engine.